phone/src/apps/twitter/components/buttons/RetweetButton.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { CircularProgress, makeStyles, Button } from '@material-ui/core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faRetweet } from '@fortawesome/free-solid-svg-icons';
+
+import Nui from '../../../../os/nui-events/utils/Nui';
+
+interface IProps {
+  tweetId: number;
+  retweetId: string;
+  isRetweet: boolean | number;
+}
+
+const useStyles = makeStyles({
+  icon: {},
+  iconFilled: {
+    color: "#00acee",
```

Comment:
There's a way implemented to use a Theme for each app.
You can see the contacts app for example which has ContactsThemeProvider and contacts.theme.ts

This way you don't need to hardcode colors and u can get them from theme.

This is really handy for example to style primary secondary buttons and textfields.

Reply:
Ok that's a great example. There are too many custom colors/css in twitter right now to do it in this PR but I added it for the two places you have commented on. I will do a follow up PR to refactor all the custom overrides into the theme.